presto-main/src/main/java/com/facebook/presto/execution/scheduler/NodeSchedulerConfig.java

```diff
@@ -76,6 +77,11 @@ public NodeSchedulerConfig setIncludeCoordinator(boolean includeCoordinator)
         return this;
     }
 
+    public int getMaxPendingSplitsPerTask()
```

Comment:
While I approve of putting this in the correct place, please make sure it does this on a separate commit (for history preservation).